### ISSUE TITLE
not translated string in install.php

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#1193]
Next hardcoded strings

![image](https://cloud.githubusercontent.com/assets/5429548/9995643/7f9c2db4-6083-11e5-980a-d3fb14ecdf02.png)

**Comment by Jimmi08** [comment]
![image](https://cloud.githubusercontent.com/assets/5429548/9995708/d8a44b44-6083-11e5-894b-1a4e068ec0a1.png)

![image](https://cloud.githubusercontent.com/assets/5429548/9995716/e3da0bf2-6083-11e5-8fe5-9af86ae3143b.png)

**Comment by Jimmi08** [comment]
install.php

![image](https://cloud.githubusercontent.com/assets/5429548/9997057/71eabef4-608a-11e5-8d1f-1a8e12a4182f.png)

![image](https://cloud.githubusercontent.com/assets/5429548/9997073/7f46a676-608a-11e5-9e91-fb81cbf0af92.png)

**Comment by Jimmi08** [comment]
Used missing lan string

![image](https://cloud.githubusercontent.com/assets/5429548/9997122/ceb865dc-608a-11e5-9418-1b64219a88f7.png)

![image](https://cloud.githubusercontent.com/assets/5429548/9997103/b8d147fc-608a-11e5-91ae-51a82b5a5a6a.png)

**Comment by Jimmi08** [comment]
Not used string (duplicate)
![image](https://cloud.githubusercontent.com/assets/5429548/9997703/f8595b96-608d-11e5-9252-a77bf76eff38.png)

This is replaced by:
![image](https://cloud.githubusercontent.com/assets/5429548/9997724/15164550-608e-11e5-91ef-2cc1068d241e.png)

**Comment by Jimmi08** [comment]
Otherwise installation process without any problems.

**Comment by tgtje** [comment]
Comment on LANINS_048 > no longer in use with that translattion the new one ( about 3 days old) is define("LANINS_048", "EXIF extension");

**Comment by Jimmi08** [comment]
I used latest languages files from github (Czech version)
![image](https://cloud.githubusercontent.com/assets/5429548/10017642/89907aac-6130-11e5-9559-4047276edddd.png)

**Comment by tgtje** [comment]
In older versions lanins 48 was decommented (not in use)<  > 3 days old now  > lanins_048 has new description.. The Czech version is 20 days, so does not have it...

**Comment by Jimmi08** [comment]
But there should be one English pack that is always update. - with watching just this one repository you could see what changed (I mean it's hard to follow whole e107 repository). The worst is that the meaning of old string was changed - you can't use translation from v1 without checking each item. But thank you for your help.

**Comment by tgtje** [comment]
I always use github base itself for these (because of date changes) much more overseeable, and also if Cameron or anyother changes the main stream...
And yes the use of v1 has its own pack (old), and of course will be overwritten by the newest, since those may become obsolete. And indeed it is a pain when using the v1 as base, hence the possibilities like those translation parts help on board.
As help you also can use a compare function of course > they should post the lines which would be non existing/different or decommented etc...

**Comment by CaMer0n** [comment]
@tgtje @Jimmi08 Is this still an issue or can it be closed now?

**Comment by tgtje** [comment]
As far as i am concerned, the issue can be closed. The installer file 
has been adjusted by one of the last commits, and as for the existing 
OLD  LAN re-use... that can happen (and probably will as older commented 
lan's will be re-used again at point in time.



---
Dit e-mailbericht is gecontroleerd op virussen met Avast antivirussoftware.
https://www.avast.com/antivirus

**Comment by Jimmi08** [comment]
Ok, I think you are right. If I find something, I create new issue. Thanks to both.
